src/main/java/org/opensrp/web/rest/TaskResource.java

```diff
@@ -281,6 +287,35 @@ public void setGson(Gson gson) {
 		modelMap.put("count", countOfTasks != null ? countOfTasks : 0);
 		return new ResponseEntity<>(modelMap, RestUtils.getJSONUTF8Headers(), HttpStatus.OK);
 	}
+
+	@RequestMapping(value = "/search/tasks-and-count", method = RequestMethod.GET, consumes = {
```

Comment:
the root of this /rest/task and therefore has lots of duplication. Can this be just `/search`

Reply:
Sure